docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-jira.md

```diff
@@ -0,0 +1,74 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-jira/
+description: Configure the Jira Service Management integration to receive notifications when your alerts are firing
+keywords:
+  - grafana
+  - alerting
+  - Jira
+  - Jira Service Management
+  - integration
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Jira Service Management
```

Comment:
```suggestion
menuTitle: Jira
```

Reply:
I tested the integration with their Cloud version. 

Jira is the brand name of the main product (Jira platform). 

Jira Service Management is a Jira App that creates Jira projects tailored for "service management and support". 

The alerting integration uses the REST API and creates "issues" in Jira projects using the REST API, supporting Jira Service Management and other types of Jira projects.

Because of this, I suggest naming the integration as `Jira` instead of `Jira Service Management`.